cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,76 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+// const faker = require('faker');
 
 describe('Settings page', () => {
-  before(() => {
-
-  });
-
-  beforeEach(() => {
-
-  });
-
   it('should provide an ability to update username', () => {
-
+    cy.createUser().then(() => {
```

Comment:
Ensure that the `cy.createUser()` command is correctly implemented in your support files. This command should handle the creation of a user and set the necessary environment variables.